app/controllers/tags_controller.rb

```diff
@@ -120,7 +120,7 @@ def children
   def rename
     status = false
 
-    Post.transaction do
+    @tag.transaction do
```

Comment:
I got `Post.transaction` from `merge` a few lines down.  Do we need to fix it there too?